bazel/envoy_internal.bzl

```diff
@@ -129,9 +129,82 @@ def envoy_copts(repository, test = False):
+    "opencensus_exporter_stackdriver": "@io_opencensus_cpp//opencensus/exporters/trace/stackdriver:stackdriver_exporter",
+    "opencensus_exporter_stdout": "@io_opencensus_cpp//opencensus/exporters/trace/stdout:stdout_exporter",
+    "opencensus_exporter_zipkin": "@io_opencensus_cpp//opencensus/exporters/trace/zipkin:zipkin_exporter",
+    "opencensus_trace": "@io_opencensus_cpp//opencensus/trace",
+    "opencensus_trace_b3": "@io_opencensus_cpp//opencensus/trace:b3",
+    "opencensus_trace_cloud_trace_context": "@io_opencensus_cpp//opencensus/trace:cloud_trace_context",
+    "opencensus_trace_grpc_trace_bin": "@io_opencensus_cpp//opencensus/trace:grpc_trace_bin",
+    "opencensus_trace_trace_context": "@io_opencensus_cpp//opencensus/trace:trace_context",
+    "opentracing": "@io_opentracing_cpp//:opentracing",
+    "path_matcher": "@grpc_httpjson_transcoding//src:path_matcher",
+    "protobuf": "@com_google_protobuf//:protobuf",
+    "protobuf_clib": "@com_google_protobuf//:protoc_lib",
+    "protoc": "@com_google_protobuf//:protoc",
+    "protocol_compiler": "@com_google_protobuf//:protoc",
+    "quiche_common_platform": "@com_github_google_quiche//:quiche_common_platform",
+    "quiche_http2_adapter": "@com_github_google_quiche//:http2_adapter",
+    "quiche_http2_hpack_decoder": "@com_github_google_quiche//:http2_hpack_decoder_hpack_decoder_lib",
+    "quiche_http2_protocol": "@com_github_google_quiche//:http2_adapter_http2_protocol",
+    "quiche_http2_test_tools": "@com_github_google_quiche//:http2_adapter_mock_http2_visitor",
+    "quiche_quic_platform": "@com_github_google_quiche//:quic_platform",
+    "quiche_quic_platform_base": "@com_github_google_quiche//:quic_platform_base",
+    "quiche_spdy_hpack": "@com_github_google_quiche//:spdy_core_hpack_hpack_lib",
+    "ragel": "@envoy//bazel/foreign_cc:ragel",
+    "re2": "@com_googlesource_code_re2//:re2",
+    "simple_lru_cache_lib": "@com_github_google_jwt_verify//:simple_lru_cache_lib",
+    "spdlog": "@com_github_gabime_spdlog//:spdlog",
+    "sqlparser": "@com_github_envoyproxy_sqlparser//:sqlparser",
+    "ssl": "@envoy//bazel:boringssl",
+    "su-exec": "@com_github_ncopa_suexec//:su-exec",
+    "tclap": "@com_github_mirror_tclap//:tclap",
+    "tcmalloc": "@com_github_google_tcmalloc//tcmalloc",
+    "tcmalloc_malloc_extension": "@com_github_google_tcmalloc//tcmalloc:malloc_extension",
+    "tcmalloc_profile_marshaler": "@com_github_google_tcmalloc//tcmalloc:profile_marshaler",
+    "uring": "@envoy//bazel/foreign_cc:liburing_linux",
+    "wamr": "@envoy//bazel/foreign_cc:wamr",
+    "wasmtime": "@com_github_wasm_c_api//:wasmtime_lib",
+    "wavm": "@envoy//bazel/foreign_cc:wavm",
+    "xxhash": "@com_github_cyan4973_xxhash//:xxhash",
+    "yaml_cpp": "@com_github_jbeder_yaml_cpp//:yaml-cpp",
+    "zlib": "@envoy//bazel/foreign_cc:zlib",
+    "zstd": "@envoy//bazel/foreign_cc:zstd",
+}
+
```

Comment:
Fun fact, this is basically what we do internally as we map into the monorepo, so this structure works well.